dsls/sql/src/test/java/org/apache/beam/dsls/sql/BeamSqlDslAggregationTest.java

```diff
@@ -257,4 +257,31 @@ public void testSessionWindow() throws Exception {
 
     pipeline.run().waitUntilFinish();
   }
+
+  @Test
+  public void testWindowOnNoneTimestampField() throws Exception {
+    exceptions.expect(IllegalStateException.class);
```

Comment:
Since these are essentially testing the user-facing interface (i.e., exceptions thrown that normal users will receive), it'd be nice to verify the message received is the correct one, e.g. checking that it contains `"Cannot apply 'TUMBLE' to arguments of type 'TUMBLE(<BIGINT>, <INTERVAL HOUR>)'"` for this one.